### PR TITLE
Add .inline-search, apply styles

### PR DESCRIPTION
**Pull request by henryhadlow**

### CODE REVIEW

File: app/assets/stylesheets/petitions/views/_search.scss [hunk 1]
```diff
@@ -3,6 +3,45 @@
...
+    @include appearance(none);
+    float: left;
+    min-width: 0;
+    margin: 0;
+    border-right: 0;
+    height: 40px;
+    padding: 6px 0;
+    width: 85%;
+    width: -webkit-calc(100% - 40px);
+    width: calc(100% - 40px);
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_search.scss]: There's a `calc` mixin we can use in `_css3.scss` in the front end toolkit

File: app/assets/stylesheets/petitions/views/_search.scss [hunk 2]
```diff
@@ -3,6 +3,45 @@
...
+    padding: 6px;
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_search.scss]: Padding has already been declared on line 17

File: app/assets/stylesheets/petitions/views/_search.scss [hunk 3]
```diff
@@ -3,6 +3,45 @@
...
+  .inline-submit {
+    float: left;
+    width: 40px;
+    height: 40px;
+    border: 0;
+    overflow: hidden;
+    text-indent: -5000px;
+    cursor: pointer;
+    color: $white;
+    background: $petitions-brand image-url("icons/icon_search-white.png") no-repeat 45% 50%;
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_search.scss]: Could use the `background-image` mixin in `assets\stylesheets\petitions\_helpers.scss`

**Comment by alexander-bobin** [comment]
Doesn't look too happy in IE7. Think it's the combo of height and padding. IE7 doesn't know what `box-sizing: border-box;` is

screen shot 2015-06-22 at 16 57 27

File: app/assets/stylesheets/petitions/views/_search.scss (hunk 3 above)
Comment (henryhadlow) [inline app/assets/stylesheets/petitions/views/_search.scss]: Classy

File: app/assets/stylesheets/petitions/views/_search.scss (hunk 1 above)
Comment (henryhadlow) [inline app/assets/stylesheets/petitions/views/_search.scss]: I have to wrap the `$calc` variable in quotes or it doesn't compile. Still cool?

**Comment by henryhadlow** [comment]
Oh man. That's why I shouldn't do frontend :(

**Comment by henryhadlow** [comment]
Fingers crossed this should do it

**Comment by alexander-bobin** [comment]
LGTM

**Pull request merged**